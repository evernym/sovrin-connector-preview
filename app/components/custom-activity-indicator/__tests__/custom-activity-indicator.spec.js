// @flow
import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import CustomActivityIndicator from '../custom-activity-indicator'

describe('<CustomActivityIndicator />', () => {
  it('should render properly and match the snapshot', () => {
    const component = renderer.create(<CustomActivityIndicator />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
