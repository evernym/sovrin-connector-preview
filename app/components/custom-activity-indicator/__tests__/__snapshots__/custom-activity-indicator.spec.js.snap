// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomActivityIndicator /> should render properly and match the snapshot 1`] = `
<View
  center={true}
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    collapsable={undefined}
    source={1}
    style={
      Object {
        "height": 30,
        "tintColor": "#777777",
        "transform": Array [
          Object {
            "rotate": "0deg",
          },
        ],
        "width": 30,
      }
    }
  />
</View>
`;
